email-worker/vitest.config.mts

```diff
@@ -0,0 +1,11 @@
+import { defineWorkersConfig } from '@cloudflare/vitest-pool-workers/config';
```

Comment:
autogenerated. can be used if we want to write tests for the email-worker in the future